src/TLuaEngine.cpp

```diff
@@ -30,12 +30,16 @@
 #include <condition_variable>
 #include <fmt/core.h>
 #include <nlohmann/json.hpp>
+#include <optional>
 #include <random>
+#include <sol/stack_core.hpp>
 #include <thread>
 #include <tuple>
 
 TLuaEngine* LuaAPI::MP::Engine;
 
+static sol::protected_function addTraceback(sol::state_view StateView, sol::protected_function RawFn);
```

Comment:
```suggestion
static sol::protected_function AddTraceback(sol::state_view StateView, sol::protected_function RawFn);
```
Let's keep with the absolutely \*\*\*\*\*\*\*\* naming scheme pls